### PR TITLE
added-temporary-comment

### PR DESCRIPTION
**Pull request by AlexaVB**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: features/step_definitions/evidence_steps.rb [hunk 1]
```diff
@@ -110,10 +110,10 @@
 Given("I have successfully submitted the evidence") do
   click_on 'Start now', visible: false
   evidence_accuracy_page.content.correct_evidence.click
-  click_on 'Next', visible: false
+  next_page
   fill_in 'Total monthly income from evidence', with: '500'
-  click_on 'Next', visible: false
-  click_on 'Next', visible: false
+  next_page
+  next_page
```
Comment (zaparka) [inline features/step_definitions/evidence_steps.rb:116]: shouldn't we check what's on that page to give capybara time to load the page?

File: features/support/page_objects/evidence_page.rb [hunk 2]
```diff
@@ -25,10 +25,10 @@ def processed_evidence
     click_on "#{reference_prefix}-000002"
     click_on 'Start now', visible: false
     evidence_accuracy_page.content.correct_evidence.click
-    click_on 'Next', visible: false
+    next_page
     fill_in 'Total monthly income from evidence', with: '500'
-    click_on 'Next', visible: false
-    click_on 'Next', visible: false
+    next_page
+    next_page
```
Comment (zaparka) [inline features/support/page_objects/evidence_page.rb:31]: same thing as comment above.

File: features/step_definitions/evidence_steps.rb (hunk 1 above)
Comment (AlexaVB) [reply features/step_definitions/evidence_steps.rb:116]: Good point Petr, I'll add something

**Comment by AlexaVB** [comment]
this was merged into another branch

**Pull request closed without merging**